lib/fizzy/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+
+add_library(fizzy)
+add_library(fizzy::fizzy ALIAS fizzy)
+
+target_sources(
```

Comment:
Using `target_sources()` instead of adding files  to `add_library` is purely pedantic at this stage. I don't believe `PRIVATE`/`PUBLIC` for sources has any useful meaning.